gamestonk_terminal/config_terminal.py

```diff
@@ -15,6 +15,30 @@
 # By default the jupyter notebook will be run on port 8888
 PAPERMILL_NOTEBOOK_REPORT_PORT = "8888"
 
+# Logging settings
+
+# 0 - INFO
+# 1 - DEBUG for terminal, INFO for libraries
+# 2 - DEBUG for terminal, DEBUG for libraries
+
+LOGGING_VERBOSITY = 2
+
+if tmp_verbosity := os.getenv("GT_LOGGING_VERBOSITY"):
```

Comment:
This is the first time I have seen a walrus operator in the wild lol. Since we have "officially" deprecated python 3.7 it's ok=)

Reply:
You probably don't have pylint enabled. If ":" is removed, pylint reports a warning of tmp_verbosity being undefined.